packages/accounts/wagmi.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "@wagmi/cli";
+import { plugingen } from "./scripts/plugingen.js";
+import { MultiOwnerPluginAbi } from "./src/msca/abis/MultiOwnerPlugin.js";
+
+export default defineConfig({
+  out: "./src/msca/plugins/multi-owner.ts",
+  contracts: [
+    {
+      name: "MultiOwnerPlugin",
+      abi: MultiOwnerPluginAbi,
+      address: "0xc56d2f3da0c2039525c95f3a71676895d9a8cae8",
```

Comment:
this is localhost. I think I'll define a `PluginGenConfig` type and come up with a better process for defining these. The idea is that you should only provide one address and chain config to use because we don't use the plugin address in code after this. The plugin address in code will actually be the same as the MSCA account itself